x/wasm/keeper/query_plugins.go

```diff
@@ -71,6 +71,7 @@ func (q QueryHandler) Query(request wasmvmtypes.QueryRequest, gasLimit uint64) (
 	}
 
 	// Issue #759 - we don't return error string for worries of non-determinism
+	moduleLogger(q.Ctx).Info("Redacting submessage error", "cause", err)
```

Comment:
This solves the issue but let's use debug level to not confuse with extra output